yaptide/application.py

```diff
@@ -16,6 +17,10 @@ def create_app():
     app = Flask(flask_name)
     logging.info("Creating Flask app %s", flask_name)
 
+    check_submodules()
+
+    from yaptide.routes.main_routes import initialize_routes
```

Comment:
Why this import is moved here ?

Reply:
When import is on top (and submodules are not loaded) there is error: "ModuleNotFoundError: No module named 'yaptide.converter'".  Thats why it should be imported after submodules are checked

Reply:
Good reason, but from reading the code itself its hard to guess that. Add a comment above explaining that `main_routes` are importing (in-directly) the converter module which is cloned as submodule. That means that lack of submodule would result in the ModuleNotFoundError.